.github/workflows/build-and-test.yml

```diff
@@ -97,7 +97,29 @@ jobs:
         with:
           os: ${{matrix.os}}
 
-      - name: Build
+      - name: Check --no-default-features
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cargo metadata --format-version 1 | \
+          jq -r '.workspace_members[] | select(contains("examples") | not)' | \
+          awk '{print $1}' | \
+          xargs -I {} cargo check -p {} --no-default-features
```

Comment:
Do we think parallelizing the `cargo check` invocations would yield any benefits?

Reply:
I actually expect it to cause more contention due to limited CPU resources and file locks.